code/game/machinery/computer/orders/order_computer/mining_order.dm

```diff
@@ -51,20 +52,7 @@
 		charge_on_purchase = FALSE,
 		manifest_can_fail = FALSE,
 	)
-	if(ltsrbt_delivered)
```

Comment:
You should remove the arg here

Reply:
What? I dont see any of your "arg" in my files

Reply:
``ltsrbt_delivered``

Reply:
oh you mean the proc hold on